res/values-tr/strings.xml

```diff
@@ -0,0 +1,23 @@
+<resources>
+    <string name="app_name">Ekstra Ayarlar</string>
+
+    <!-- Strings related to Settings -->
+    <string name="pref_description_adbonswitch">Ağ (WiFi, USB ağ paylaşımı, WiFi Hotspot) üzerinden TCP/IP hata ayıklamayı açar. Bu ayar telefon yeniden başlatıldığı zaman sıfırlanır.</string>
+    <string name="pref_description_adbonswitchdialog">UYARI: Ağ üzerinden ADB açık iken telefonunuz bağlı olduğunuz ağlarda saldırılara karşı açık olabilir!\n\nBu özelliği sadece güvendiğiniz ağlar üzerinde kullanın.\nBu özelliği gerçekten etkinleştirmek istiyor musunuz?</string>
+    <string name="pref_title_adbonswitch">Ağ üzerinden ADB</string>
+    <string name="pref_description_drsdialog">Dinamik Çözünürlük ayarını etkinleştirmek için arayüz yeniden başlatılacak.\n\nDevam etmek istiyor musunuz?</string>
+    <string name="pref_title_dynres_switch">Dinamik Çözünürlük Ayarı</string>
+    <string name="pref_title_dispcal_switch">Beyaz Nokta Kalibrasyonu</string>
+    <string name="error_connect_to_wifi">Hata: Bir WiFi ağına bağlı olmanız gerekiyor!</string>
+    <string name="pref_title_glove_mode_switch">Eldiven modu</string>
+    <string name="pref_title_glove_mode_switch_summary">Dokunmatik hassasiyetini artırır</string>
+    <string name="dialog_reboot_required">Yeniden başlatma gerekiyor</string>
+    <string name="dialog_reboot_required_summary">Değişikliklerin uygulanması için cihazın yeniden başlatılması gerekiyor!</string>   
+    <string name="dialog_enable_glove_mode">Eldiven modu dokunmaların doğruluğunu azaltabilir. <i>Dokunmayı göster</i> ayarı eldiven modunun açık olup olmadığından haberdar olmanız için etkinleştirilecek. <i>Geliştirici seçenekleri</i> menüsünden devre dışı bırakabilirsiniz.</string>
+    <string name="dialog_disable_glove_mode">Eldiven modunu kapatmak Geliştirici seçenekleri&apos;ndeki <i>Dokunmayı göster</i> ayarını kapatır.</string>
```

Comment:
The apostrophe needs to be escaped with `\'` instead:
```
[ 10% 1/10] AAPT2 compile out/target/common/obj/APPS/ExtendedSettings_intermediates/flat-res/packages/apps/ExtendedSettings/res/values-t
FAILED: out/target/common/obj/APPS/ExtendedSettings_intermediates/flat-res/packages/apps/ExtendedSettings/res/values-tr_strings.arsc.flat
/bin/bash -c "out/host/linux-x86/bin/aapt2 compile -o out/target/common/obj/APPS/ExtendedSettings_intermediates/flat-res/packages/apps/ExtendedSettings/res/ --pseudo-localize --legacy packages/apps/ExtendedSettings/res/values-tr/strings.xml"
packages/apps/ExtendedSettings/res/values-tr/strings.xml:17: error: unescaped apostrophe in string
"Eldiven modunu kapatmak Geliştirici seçenekleri'ndeki ".
packages/apps/ExtendedSettings/res/values-tr/strings.xml:17: error: not a valid string.
packages/apps/ExtendedSettings/res/values-tr/strings.xml: error: file failed to compile.
16:28:51 ninja failed with: exit status 1
```

Reply:
Yeah i didnt build it. Im almost Felix now.
Will fix really quick.